server/utils/github_helper.py

```diff
@@ -65,16 +67,15 @@ def fetch_method_from_repo(node):
             app_auth = auth.get_installation_auth(response.json()["id"])
             github = Github(auth=app_auth)
             repo = github.get_repo(repo_name)
-            file_contents = repo.get_contents(file_path.replace("\\","/"), ref=branch_name)
```

Comment:
this change is to handle filenames in windows, we should not revert this especially since the focus here is local dev